package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "jest",
-    "webpack": "webpack"
+    "webpack": "webpack",
+    "lint": "./node_modules/.bin/eslint ./"
```

Comment:
Is that the recommended way to do it? I think you can use npx instead that does exactly this...